.github/unittest/linux_olddeps/scripts_gym_0_13/environment.yml

```diff
@@ -25,3 +25,4 @@ dependencies:
     - patchelf
     - pyopengl==3.1.4
     - ray<2.8.0
+    - torchsnapshot
```

Comment:
Nope let's avoid installing it everywhere, let's just skip tests where it is not installed already